Sources/Testing/CMakeLists.txt

```diff
@@ -96,6 +96,8 @@ add_library(Testing
   Traits/Trait.swift)
 target_link_libraries(Testing PRIVATE
   _TestingInternals)
+target_link_libraries(Testing PUBLIC
```

Comment:
Will this fail if for some reason we _can't_ find Foundation?

Reply:
I think it should fail with linker errors.